song-server/src/test/java/bio/overture/song/server/controller/AnalysisTypeControllerTest.java

```diff
@@ -849,7 +850,7 @@ public static List<AnalysisType> generateData(
             i -> {
               val name = names.get(i % repeats);
               val schema = generateRandomRegistrationPayload(randomGenerator);
-              val out = analysisTypeService.register(name, schema);
+              val out = analysisTypeService.register(name,new AnalysisTypeOptions(), schema);
```

Comment:
Looks like code formatting hasn't been run on this line, no space between comma and next argument.